src/routes/adminPanel/adminPanelTable/adminPanelTable.styles.tsx

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { color } from '../../../theme';
+import { regularBlack } from '../../../theme/border';
+
+export const HEIGHT = '40px';
+
+export const Styles = styled.div`
+  padding: 1rem;
+  overflow: scroll;
+  height: 100%;
+
+  table {
+    border-spacing: 0;
+    border: ${regularBlack};
+
+    tr {
+      :last-child {
+        td {
+          border-bottom: 0;
+        }
+      }
+    }
+
+    th,
+    td {
+      margin: 0;
+      padding: 0.5rem;
+      border-bottom: ${regularBlack};
+      border-right: ${regularBlack};
```

Comment:
it would be more consistent when you will use `color, border from theme`, and then `border.regularBlack`.